src/briefcase/console.py

```diff
@@ -250,6 +250,75 @@ def export_log(self):
         """Export the text of the entire log; the log is also cleared."""
         return self._log_impl.export_text()
 
+    def warning_banner(
+        self,
+        message: str,
+        title: str = "",
+        width=80,
+        border_char="*",
+        padding=2,
```

Comment:
Missing type annotations:
```suggestion
        width: int = 80,
        border_char: str = "*",
        padding: int = 2,
```